### PR TITLE
Refactor adapt 2.0.0rc2

### PR DESCRIPTION
**Pull request by alanthssss**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,57 +1,23 @@
...
+        from jina.helloworld.fashion.helper import load_mnist
+        from jina.helloworld.fashion.executors import MyEncoder, MyIndexer
+        from jina.types.document.generators import from_ndarray
 
         from pkg_resources import resource_filename
 
         err_msg = ''
         index_size = 60000
-        query_size = 4096
+        query_size = 10000
```
Comment (alanthssss) [inline app.py]: image

**Review by alanthssss — COMMENTED** [review]
I find that the query_size may be 10000.

File: app.py (hunk 1 above)
Comment (alanthssss) [reply app.py]:
```python
'index_time': index_time,
'query_time': query_time,
'index_qps': index_size / index_time,
'query_qps': query_size / query_time,
```
`index_size` is the number of all index data, but `query_size` is not the number of all query data.

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**